.ci/install_kata_kernel.sh

```diff
@@ -13,80 +13,72 @@ set -e
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 
-repo_owner="clearcontainers"
-repo_name="linux"
-
-linux_releases_url="https://github.com/${repo_owner}/${repo_name}/releases"
-#fake repository dir to query kernel version from remote
-fake_repo_dir=$(mktemp -t -d kata-kernel.XXXX)
-
-function cleanup {
-	rm  -rf "${fake_repo_dir}"
+# FIXME: waiting for https://github.com/kata-containers/packaging/pull/18
+repo_name="kata-packaging"
+repo_owner="devimc"
+############
+kata_kernel_dir="/usr/share/kata-containers"
+kernel_arch="x86_64"
```

Comment:
Can we just make this `kernel_arch=$(arch)`? We can handle unknown architectures in `build_and_install_kernel()`.